src/langs/json/en.json

```
@@ -383,7 +384,7 @@
     "fileLimitWarning": "You can only upload {limit} files at once.",
     "beginUpload": "Begin Upload",
     "selectFiles": "Select files",
-    "dragDrop": "Select, drag and drop, or paste files to upload",
+    "dragDrop": "Select, paste or drag and drop files to upload",
```

For consistency with other d-n-d text below (413):
```suggestion
    "dragDrop": "Select, paste or drag-and-drop files to upload",
```